test/e2e/page-objects/CheckoutOverview.js

```diff
@@ -0,0 +1,91 @@
+import Base from './Base';
+
+const SCREEN_SELECTOR = '#checkout_summary_container';
+
+class CheckoutOverview extends Base {
```

Comment:
@wswebcreation hmm, same here. This and `CartOverview` I imagine to be the same page objects. Looking at the Selector, I see `#checkout_summary_container`, so if this is `CheckoutSummary` page then I understand what it is

Reply:
tnx @nadvolod, I can rename it, no problem, shall I then also rename the cart to CartSummary?

Reply:
@wswebcreation I'm not sure I 100% understand your question?

Reply:
in general page object class names should be as close as possible to the actual title of the page, or the name on the container of the modal, etc, etc. You want to be able to read the action on the page object as if it were the words written in a cucumber feature file. ;-)

Reply:
Renamed it with my last push, hope it's better now